src/topp/ConsensusID.cpp

```diff
@@ -106,7 +108,7 @@ using namespace std;
     <B>File types:</B>
 
     Different input files types are supported:
-    @li idXML: A file containing multiple identification runs, typically from different search engines. Use @ref TOPP_IDMerger to merge individual idXML files from different search runs into one. During the ConsensusID analysis, the identification results will be grouped according to their originating MS2 spectra, based on retention time and precursor m/z information (see parameters @p rt_delta and @p mz_delta). One consensus identification will be generated for each group.
+    @li idXML: A file containing multiple identification runs, typically from different search engines. Use @ref TOPP_IDMerger to merge individual idXML files from different search runs into one. During the ConsensusID analysis, the identification results will be grouped according to their originating MS2 spectra, based on retention time and precursor m/z information (see parameters @p rt_delta and @p mz_delta). One consensus identification will be generated for each group. With the per_spectrum flag you can also input multiple idXML files. A consensus will be made per combination of originating mzml file and spectrum_ref.
```

Comment:
does this check the referenced mzML is the same?

Reply:
No, it just groups per mzml+spectrumref and creates a new run per mzml.
Was thinking about extending to mzml_list (in case someone wants to consensusID a merge across fractions.

Reply:
not sure if needed at this point